inference-engine/cmake/dependencies.cmake

```diff
@@ -147,6 +147,8 @@ if (ENABLE_OPENCV)
             set(OPENCV_SUFFIX "ubuntu16")
         elseif (${LINUX_OS_NAME} STREQUAL "Ubuntu 18.04")
             set(OPENCV_SUFFIX "ubuntu18")
+        else()
+            set(OPENCV_SUFFIX "ubuntu18")
```

Comment:
This is a dangerous change.

I believe error message should be emitted instead with suggestion of forcing OPENCV_SUFFIX value or changing other options (like disabling OpenCV or enabling own binaries).

Reply:
Agree. So far I have to use docker build dldt based on Ubuntu 19.10 because of some limitation. So I need curl this PR in docker file and patch it. I can upload another PR for emitting error message.  Thanks.

Reply:
Now we have error message that OpenCV is not found https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/cmake/dependencies.cmake#L234-L239